Alignment/APEEstimation/test/SkimProducer/skimProducer_cfg.py

```diff
@@ -161,9 +153,6 @@
     process.MuSkim.src = 'TrackList'
     process.TriggerSelectionSequence *= process.TrackList
 
-import Alignment.CommonAlignment.tools.trackselectionRefitting as trackselRefit
```

Comment:
You are right. I confused this selection with a refit we do later on and thought it was obsolete.

I reintroduced it and put the trackselRefit sequence back into the path (where it was not included in the past, which might not have been ideal). 

The actual impact on the APEs should be small since the tool includes another refit (https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_X/Alignment/APEEstimation/test/cfgTemplate/apeEstimator_cfg.py#L166), but it might speed up the process since bad tracks are already discarded in the skimming step and not used again in the refit for iteration of the APE tool